boards/nrf52-minidev/Makefile.include

```diff
@@ -0,0 +1,29 @@
+# define the cpu used by the nRF52 DK
+export CPU = nrf52
+export CPU_MODEL = nrf52xxaa
+
+# set default port depending on operating system
+PORT_LINUX ?= /dev/ttyUSB0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
+
+# setup the boards dependencies
+include $(RIOTBOARD)/$(BOARD)/Makefile.dep
+
+# Define the flash-tool and default port:
+export PROGRAMMER ?= openocd
+
+ifeq ($(PROGRAMMER),openocd)
+else ifeq ($(PROGRAMMER),jlink)
+	# setup JLink for flashing
+	export JLINK_DEVICE := nrf52
+	# special options when using SoftDevice
+	ifneq (,$(filter nordic_softdevice_ble,$(USEPKG)))
+		export JLINK_PRE_FLASH := loadfile $(BINDIR)/softdevice.hex
```

Comment:
The jlink stuff is taken from nrf52dk. I cannot check the functionality with a jlink adapter.